mods/ra/rules/world.yaml

```diff
@@ -4,6 +4,8 @@
 	ActorMap:
 	ScreenMap:
 	MusicPlaylist:
+		StartingMusic: intro
```

Comment:
We only have hell march in the initial scores.mix, but I don't see how that is a problem here.